core/operation/src/test/java/uk/gov/gchq/gaffer/named/operation/AddNamedOperationTest.java

```diff
@@ -52,13 +43,19 @@
 
     @Override
     public void shouldJsonSerialiseAndDeserialise() {
```

Comment:
Thanks for updating this test. Could we add another one to check backwards compatibility? ie serialising with no options

Reply:
Done, I've set the options to null and removed them from the expected result.